tox.ini

```diff
@@ -29,6 +29,8 @@ per-file-ignores =
     docs/source/conf.py: E501
     # Typedef files are formatted differently
     *.pyi: E301, E302, F401
+    # D414 is needed due to a bug in pydocstyle that sees the returntype `Parameters` as nested sections
```

Comment:
'ReturnType' in the context of Javascript, 'return type' otherwise.

Reply:
```suggestion
    # D414 is needed due to a bug in pydocstyle that sees the return type `Parameters` as nested sections
```